std_msgs/Header header
string[] value_names
string[] pose_names
string[] vector_names
string[] twist_names
float64[] values
geometry_msgs/Pose[] poses
geometry_msgs/Vector3[] vectors
geometry_msgs/Twist[] twists
